flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java

```diff
@@ -65,8 +65,7 @@ public class MySqlConnectorITCase extends MySqlSourceTestBase {
     private static final String TEST_USER = "mysqluser";
     private static final String TEST_PASSWORD = "mysqlpw";
 
-    private static final MySqlContainer MYSQL8_CONTAINER =
-            (MySqlContainer) createMySqlContainer(MySqlVersion.V8_0).withExposedPorts(3307);
+    private static final MySqlContainer MYSQL8_CONTAINER = createMySqlContainer(MySqlVersion.V8_0);
```

Comment:
Why need change this line?

Reply:
I changed this line when I fixed a falling test, where this container was used. This port exposure does not make sense, but only confuses the reader of the test.

Reply:
The default port 3306 has been used in the `MySqlSourceTestBase` by the `MYSQL_CONTAINER` whose mysql version is 5.7. So we need to keep this setting.

Reply:
As I said it confuses the reader of the test. In tests with `MYSQL8_CONTAINER`, when we get database port in table options via `MySqlContainer.getDatabasePort()` we get mapped port for a given port exposed by the container (it's always `MYSQL_PORT`(3306), even before my changes). So, there would be no port collision with `MYSQL_CONTAINER`, because they both use 3306 **inside** of the container, but mapped **host** ports are different.